todo-list-ui/src/app/todo-modal/todo-modal.component.html

```diff
@@ -0,0 +1,39 @@
+<ng-template #content let-modal>
+  <div class="modal-header">
+    <h4 class="modal-title" id="todo-moda-title">Add todo</h4>
+    <button
+      type="button"
+      class="btn-close"
+      aria-label="Close"
+      (click)="modal.dismiss('Cross click')"
+    ></button>
+  </div>
+  <div class="modal-body">
+    <form>
+      <div class="mb-3">
+        <input
+          id="inputTodo"
+          class="form-control"
+          type="text"
+          placeholder="Example: Do lunch"
+          [formControl]="inputTodo"
+          required
+        />
+      </div>
+    </form>
+  </div>
+  <div class="modal-footer">
+    <button
+      type="button"
+      class="btn btn-primary"
+      [disabled]="!inputTodo.value"
```

Comment:
Instead of manually checking if the input is empty. Look into Angular validations https://angular.dev/guide/forms/form-validation#built-in-validator-functions, then you would change this line: `[disabled]="inputTodo.invalid"`